tests/components/geniushub/conftest.py

```diff
@@ -38,6 +39,26 @@ def mock_geniushub_client() -> Generator[AsyncMock]:
         yield client
 
 
+@pytest.fixture
+def mock_geniushub_cloud() -> Generator[AsyncMock]:
```

Comment:
I don't think you can do `await client` 
```suggestion
def mock_geniushub_cloud() -> Generator[MagicMock]:
```